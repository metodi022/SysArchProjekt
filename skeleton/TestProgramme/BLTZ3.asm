lui $1, 0x101
bltz $1, end
ori $1, 0
end:
ori $1, 1