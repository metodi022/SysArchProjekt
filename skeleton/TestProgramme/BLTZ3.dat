3c010101
04200001
34210000
34210001
