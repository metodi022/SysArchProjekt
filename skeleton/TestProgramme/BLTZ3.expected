@1 01010001
